R/build.r

```diff
@@ -308,11 +308,14 @@ build_site_external <- function(pkg = ".",
     preview = FALSE,
     new_process = FALSE,
     crayon_enabled = crayon::has_color(),
-    crayon_colors = crayon::num_colors()
+    crayon_colors = crayon::num_colors(),
+    pkgdown_internet = has_internet()
   )
   callr::r(
-    function(..., crayon_enabled, crayon_colors) {
-      options(crayon.enabled = crayon_enabled, crayon.colors = crayon_colors)
+    function(..., crayon_enabled, crayon_colors, pkgdown_internet) {
+      options(crayon.enabled = crayon_enabled,
```

Comment:
Can you please indent this with tidyverse style? http://style.tidyverse.org/syntax.html#long-lines

Reply:
Oh sorry. I used styler 📦 to check the style thinking it would find everything. Not long lines apparently.